src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-overview-dashboard/rgw-overview-dashboard.component.html

```diff
@@ -92,20 +92,185 @@
       <cd-card cardTitle="Used Capacity"
                i18n-title
                class="col-sm-2 d-flex w-100 h-50 pb-3"
-               aria-label="Details card">
+               aria-label="Used Capacity">
         <span  class="ms-4 me-4 text-center">
           <h1>{{ totalPoolUsedBytes | dimlessBinary}}</h1>
         </span>
       </cd-card>
       <cd-card cardTitle="Avg Object Size"
                i18n-title
                class="col-sm-2 d-flex w-100 h-50 pt-3"
-               aria-label="Details card">
+               aria-label="Avg Object Size">
         <span class="ms-4 me-4 text-center">
           <h1>{{ averageObjectSize | dimlessBinary}}</h1>
         </span>
       </cd-card>
     </div>
   </div>
+  <div class="row pt-4">
+    <cd-card cardTitle="Multisite Sync Status"
+             i18n-title>
+      <div *ngIf="!showMultisiteCard"
+           class="text-center">
+           No Data!
+      </div>
+      <div class="row"
+           *ngIf="showMultisiteCard">
+        <div class="row pt-4">
+          <cd-card cardTitle="Primary Source Zone"
+                   class="col-lg-3 d-flex justify-content-center align-primary-zone">
+            <span *ngIf="loading"
+                   class="d-flex justify-content-center">
+            <i [ngClass]="[icons.large3x, icons.spinner, icons.spin]"></i></span>
+            <span *ngIf="!loading"
+                  class="d-flex justify-content-center">
+              <ul class="pb-5">
+                <li><i [ngClass]="[icons.large2x, icons.reweight]"
+                       class="pt-2"></i></li>
+                <li class="badge badge-info mt-2">{{realm}}</li>
+                <li><i [ngClass]="[icons.large2x, icons.down]"
+                       class="mt-2"></i></li>
+                <li><i [ngClass]="[icons.large2x, icons.cubes]"
+                       class="mt-2"></i></li>
+                <p class="badge badge-info mt-2">{{zonegroup}}</p>
+                <li><i [ngClass]="[icons.large2x, icons.down]"
+                       class="mt-2"></i></li>
+                <li><i [ngClass]="[icons.large2x, icons.deploy]"
+                       class="mt-2"></i></li>
+                <li class="badge badge-info mt-2">{{zone}}</li>
+              </ul>
+            </span>
+          </cd-card>
+          <div class="col-lg-9">
+            <cd-card cardTitle="Source Zones"
+                     class="d-flex h-100">
+              <span *ngIf="loading"
+                     class="d-flex justify-content-center">
+              <i [ngClass]="[icons.large3x, icons.spinner, icons.spin]"></i></span>
+              <div class="row"
+                   *ngIf="!loading">
+                <cd-card  cardTitle="{{zone.name}}"
+                          *ngFor="let zone of replicaZonesInfo"
```

Comment:
this should solve the issues like `tooltip` gets autoclosed after 5 seconds because of the refresh. 
```suggestion
                          *ngFor="let zone of replicaZonesInfo;  trackBy: trackByFn"
```

and you should have a `trackByFn` which tracks the zone.
```
  trackByFn(zone: any) {
    return zone;
  }
```

Reply:
Done. thanks @nizamial09